docs/components/generic/_index.md

```diff
@@ -38,23 +39,10 @@ If you want to use most of an existing API but need just a few other functions,
 
 ## Supported models
 
-Before creating a new generic component, check whether one of the following [modular resources](#modular-resources) supports your use case.
+{{<resources api="rdk:component:generic" type="generic">}}
```

Comment:
"To use your generic, check whether one of the following models supports your generic:" doesn't make much sense. Can we manually override to make this "To use your generic **component**, check whether one of the following models supports your **component**:"? (with no "generic" in the second instance)

Reply:
why don't we add component everywhere instead

Reply:
I guess if it's hard to do the manual override that would work yeah!